README.md

```diff
@@ -16,12 +16,50 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
-npm install @stoplight/spectral
+### Local Installation
+
+```bash
+npm install --save-dev @stoplight/spectral
+```
+
+### Global Installation
+
+```bash
+npm install -g @stoplight/spectral
 ```
 
 Supports Node v8.3+ and modern browsers.
 
+### Executable binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for all major platforms:
+
+- x64 Windows
+- x64 MacOS
+- x64 Linux
+
+You can find them [here](https://github.com/stoplightio/spectral/releases).
+Once downloaded, you can proceed with the standard procedure for running any CLI tool.
+
+```bash
+./spectral-macos lint petstore.yaml
+```
+
+Note, the binaries are *not* auto-updatable, therefore you will need to download a new version on your own.
+
+#### Installing binaries system-wide
+
+##### Linux
+
+```bash
+sudo mv ./spectral-linux /usr/local/bin/spectral
+```
+
+You may need to restart your terminal.
+Now, `spectral` command will be accessible in your terminal.
+
+Head over to [releases](https://github.com/stoplightio/spectral/releases) for the latest binaries.
```

Comment:
Is this true? Binaries will be in releases?

Reply:
For 2.0.x I'll add them automatically. There is a US to integrate the build process with CI supposed to be done during the current sprint, so I think it's likely to be true in general.

Reply:
Yeah we will add the files to the release ourselves then automate it later